spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -479,7 +479,7 @@ private void setupConfForPySpark(SparkConf conf) {
 
     //Only one of py4j-0.9-src.zip and py4j-0.8.2.1-src.zip should exist
     String[] pythonLibs = new String[]{"pyspark.zip", "py4j-0.9-src.zip", "py4j-0.8.2.1-src.zip",
-      "py4j-0.10.1-src.zip"};
+      "py4j-0.10.3-src.zip"};
```

Comment:
You means that add "py4j-0.10.3-src.zip" after "py4j-0.10.1-src.zip"? Can it confuse the logic below? 

`  
    // Distribute two libraries(pyspark.zip and py4j-*.zip) to workers
    // when spark version is less than or equal to 1.4.1
    if (pythonLibUris.size() == 2) {
`

If this semantics is the same as before adding. I will follow your comment to modify.